apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/Constants.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+namespace Google.Cloud.Spanner.Data.CommonTesting;
+
+public static class Constants
+{
+    public const string Category = nameof(Category);
+    public const string GSQL = nameof(GSQL);
+    public const string PostgreSQL = nameof(PostgreSQL);
+    public const string Admin = nameof(Admin);
+    public const string SkipOnEmulator = nameof(SkipOnEmulator);
+    public const string Yes = nameof(Yes);
```

Comment:
What are these Yes and No for? (It's not obvious to me what `[Trait(Constants.SkipOnEmulator, Constants.Yes)]` means for example.

Doc comments on all of these would be useful.